test/plugins/foreign/insert-nested-inline.js

```diff
@@ -0,0 +1,23 @@
+/** @jsx h */
+
+export default editor => editor.wrapInline({ type: 'term' })
+
+export const input = <value>
+    <document>
+        <p>
+            <text/>
+            <foreign>Foreign <anchor/>term<focus/> text</foreign>
+            <text/>
+        </p>
+    </document>
+</value>
+
+export const output = <value>
+    <document>
+        <p>
+            <text/>
+            <foreign>Foreign <term>term</term><cursor/> text</foreign>
```

Comment:
Selection should stay around the term.

Reply:
This is current behavior of slate. Would you like me to remove this test, comment it out or do something else?